lib/instrumentation/index.js

```diff
@@ -59,8 +59,7 @@ function Instrumentation (agent) {
   this._hook = null // this._hook is only exposed for testing purposes
   this._started = false
   this._runCtxMgr = null
-
-  this._log = agent.logger.child({ 'event.module': 'instrumentation' })
+  this._log = agent.logger
```

Comment:
REVIEW NOTE: There was no value in the `event.module` field in the log records, so I dropped it rather than having to reproduce it in the `Instrumentation.prototype.start` change below.